Homework#1/number_of_desks.py

```diff
@@ -0,0 +1,21 @@
+"""В некоторой школе решили набирать три новых математических класса и
+оборудовать кабинеты для них новыми партами. За каждой партой может сидеть два учащихся.
+Известно количество учащихся в каждом из трех классов.
+Выведите наименьшее число парт, которое нужно приобрести для них."""
+
+import math
+
+#Формула для вычисления количества парт
+def quantity_desks(x):
```

Comment:
Pls add type notation

```suggestion
def quantity_desks(x: int) -> int:
```

Reply:
Did I understand correctly that in all tasks I need to add an annotation of the type?

Reply:
Yes

Reply:
done